app/src/main/java/com/jerboa/ui/components/common/TextBadge.kt

```diff
@@ -52,13 +53,13 @@ fun TextBadge(
 /**
  * Displays activitypub items (communities, users), with a smaller @instance shown
  */
-@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun ItemAndInstanceTitle(
     modifier: Modifier = Modifier,
     title: String,
     actorId: String?,
     local: Boolean,
+    onClick: (() -> Unit)?,
```

Comment:
There's no need to make these optional IMO. If its called as `onClick = {}`, it at least forces us to look at each case where its called.

Reply:
No it is important this is optional, We can't add a clickable modifier for these. Else it wouldn't propagate the click to he parent clickable modifier. This empty onClick handler would capture the click

Reply:
Ah I see.